modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/gradle.properties.mustache

```diff
@@ -1,4 +1,4 @@
-{{#generateRoomModels}}
 android.useAndroidX=true
+{{#generateRoomModels}}
```

Comment:
jvm-volley refuses to build without `android.useAndroidX=true`, even if generateRoomModels is false